javascript-modules-engine/pom.xml

```diff
@@ -230,7 +230,7 @@
 
                     <!-- hack: until we find a cleaner solution, run a 'yarn install' on the jahia-module project (used for testing) as part of Maven build -->
```

Comment:
let's completely remove this section if no longer needed

Reply:
Done!